frontend/packages/ez-transactions/package.json

```diff
@@ -16,7 +16,7 @@
     "react-dom": "18.2.0"
   },
   "dependencies": {
-    "@gear-js/vara-ui": "0.0.10",
+    "@gear-js/vara-ui": "0.0.6",
```

Comment:
What's the point of downgrade?

Reply:
All dapps use 0.0.6 version. Because different versions are used, some styles are not loaded